Assets/MixedRealityToolkit.Extensions/SceneTransitionService/Scripts/LoadContentScene.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: this has been reduced in length:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
Updated license header.